react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,30 @@
+#root {
+  width: 160px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+.fa-times {
+  color: red;
+}
+
+.fa-check {
+  color: green;
+}
```

Comment:
You don't want to change the behavior of library classes directly. This can cause issues and confusion in other parts of your project, especially if you have other people working on the project with you (usually the case). Someone might try to use `fa-times` somewhere else and would expect it to be black by default. So a better solution would be to make stand alone classes to change the text color.

```CSS
.text-red {
  color: red;
}

.text-green {
  color: green;
}
```